packages/ui/src/components/organisms/SKUMatrix/styles.scss

```diff
@@ -0,0 +1,162 @@
+[data-fs-sku-matrix-sidebar] {
+  // --------------------------------------------------------
+  // Design Tokens for SKU Matrix Sidebar
+  // --------------------------------------------------------
+
+  // Default properties
+
+  // Background
+  --fs-sku-matrix-sidebar-bkg-color				: var(--fs-color-neutral-0);
+
+  // Title
+  --fs-sku-matrix-sidebar-title-size				: var(--fs-text-size-6);
+  --fs-sku-matrix-sidebar-title-text-weight			: var(--fs-text-weight-semibold);
+
+  // Cell
+  --fs-sku-matrix-sidebar-table-cell-font-size			: var(--fs-text-size-0);
+  --fs-sku-matrix-sidebar-table-cell-text-weight		: var(--fs-text-weight-medium);
+
+  // Font weight
+  --fs-sku-matrix-sidebar-text-weight-medium			: var(--fs-text-weight-medium);
+  --fs-sku-matrix-sidebar-text-weight-semibold			: var(--fs-text-weight-semibold);
+
+  // Spacing
+  --fs-sku-matrix-sidebar-spacing-2				: var(--fs-spacing-2);
+  --fs-sku-matrix-sidebar-spacing-3				: var(--fs-spacing-3);
+  --fs-sku-matrix-sidebar-spacing-4				: var(--fs-spacing-4);
+  --fs-sku-matrix-sidebar-spacing-8				: var(--fs-spacing-8);
```

Comment:
suggestion: I think we may not need all these tokens. I’d suggest renaming `--fs-sku-matrix-sidebar-spacing-8` to` --fs-sku-matrix-sidebar-padding` and using the other values directly without creating additional tokens.